pkg/webui/console/views/gateway-overview/gateway-overview.js

```diff
@@ -33,16 +38,29 @@ import GatewayConnection from '@console/containers/gateway-connection'
 
 import withFeatureRequirement from '@console/lib/components/with-feature-requirement'
 
+import convertStrToDataUri from '@ttn-lw/lib/convert-str-to-data-uri'
+import downloadObjectAsFile from '@ttn-lw/lib/download-object-as-file'
+import { selectGsConfig } from '@ttn-lw/lib/selectors/env'
 import PropTypes from '@ttn-lw/lib/prop-types'
 import sharedMessages from '@ttn-lw/lib/shared-messages'
 
 import { mayViewGatewayInfo } from '@console/lib/feature-checks'
 
+const m = defineMessages({
+  downloadGlobalConf: 'Download global_conf.json',
+  globalConf: 'Global configuration',
+})
+
+@injectIntl
 @withRequest(({ gtwId, loadData }) => loadData(gtwId), () => false)
 @withFeatureRequirement(mayViewGatewayInfo, {
   redirect: '/',
 })
 export default class GatewayOverview extends React.Component {
+  state = {
+    globalConf: null,
```

Comment:
Use redux store

Reply:
it was too complicated for me to figure out how to dispatch in the redux store as a callback. will leave it for now and maybe look at it later